packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -63,7 +64,7 @@ class MojoAssetBundle extends AssetBundle {
   }
 
   AssetBundleProxy _bundle;
-  Map<String, ImageResource> _imageCache = new Map<String, ImageResource>();
+  Map<String, ui.Image> _imageCache = new Map<String, ui.Image>();
```

Comment:
I don't think it makes sense to have a map of strings to ui.Image objects. Consider multipart push jpegs or animated gifs. They will probably have multiple ui.Image objects for a single key.